### PR TITLE
docs: update agent updates

### PR DESCRIPTION
**Pull request by stevenGravy**
Based on recent feedback:
- more explicit on Teleport Enterprise version needed for agent. Wasn't clear the agent needs to be Teleport Enterprise.
- Note to not have `https://` as a prefix for systemd address.
- provide variables where able.

### CODE REVIEW

File: docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx [hunk 1]
```diff
@@ -111,6 +122,9 @@ not be exported yet.
 
 <TabItem label="teleport-kube-agent chart" options="Self-hosted">
 
+Confirm you are using the enterprise image. The `enterprise` value setting
```
Comment (ptgott) [inline docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx:125]:
```suggestion
Confirm you are using the Teleport Enterprise image. The `enterprise` value setting
```

File: docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx [hunk 2]
```diff
@@ -111,6 +122,9 @@ not be exported yet.
...
+should have been set to `true` for the helm chart installation.
```
Comment (ptgott) [inline docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx:126]:
```suggestion
should have been set to `true` for the Helm chart installation.
```

File: docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx [hunk 3]
```diff
@@ -143,6 +157,13 @@ $ kubectl logs <your-agent-release>-updater
 </TabItem>
 <TabItem label="systemd " options="Teleport Cloud">
 
+Confirm you have the Teleport Enterprise version installed.
```
Comment (ptgott) [inline docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx:160]:
```suggestion
Confirm you have the Teleport Enterprise edition installed.
```

File: docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx [hunk 4]
```diff
@@ -184,6 +205,9 @@ not be exported yet.
 </TabItem>
 <TabItem label="teleport-kube-agent chart " options="Teleport Cloud">
 
+Confirm you are using the enterprise image. The `enterprise` value setting
```
Comment (ptgott) [inline docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx:208]:
```suggestion
Confirm you are using the Teleport Enterprise image. The `enterprise` value setting
```

File: docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx [hunk 5]
```diff
@@ -184,6 +205,9 @@ not be exported yet.
...
+should have been set to `true` for the helm chart installation.
```
Comment (ptgott) [inline docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx:209]:
```suggestion
should have been set to `true` for the Helm chart installation.
```

File: docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx [hunk 6]
```diff
@@ -69,16 +76,20 @@ Else, you can skip this step:
 
 ```code
 $ sudo touch /etc/teleport-upgrade.d/schedule
-$ sudo chown <your-teleport-user> /etc/teleport-upgrade.d/schedule
+$ sudo chown <Var name="your-teleport-user" /> /etc/teleport-upgrade.d/schedule
```
Comment (zmb3) [inline docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx:79]: I thought you couldn't use vars in code blocks?

Comment (stevenGravy) [reply docs/pages/management/operations/enroll-agent-into-automatic-updates.mdx:79]: You can.  You can't in other types like `yaml`. 

image

**Review by hugoShaka — APPROVED** [review]

**Review by ptgott — APPROVED** [review]

**Pull request merged**